gdx/src/com/badlogic/gdx/maps/tiled/renderers/OrthoCachedTiledMapRenderer.java

```diff
@@ -209,7 +209,9 @@ public void renderObject (MapObject object) {
 
 	@Override
 	public void renderTileLayer (TiledMapTileLayer layer) {
-		final float color = Color.toFloatBits(1, 1, 1, layer.getOpacity());
+
+		final float color = Color.toFloatBits(1 * layer.getCombinedTintColor().r, 1 * layer.getCombinedTintColor().g,
+			1 * layer.getCombinedTintColor().b, layer.getOpacity() * layer.getCombinedTintColor().a);
```

Comment:
Multiplying by 1 looks a bit odd to me...

Reply:
Yep, this does look odd. More of an oversight on my part since the OrthoCachedTiledMapRenderer is the odd man out when it comes to all the Map Renderers. It originally passed in all 1s as it's the only renderer which didn't use the spritebatch color.
This will be cleaned up, when I push the next update.